include/grpcpp/impl/codegen/call_op_set.h

```diff
@@ -285,7 +285,7 @@ class CallOpSendInitialMetadata {
 
 class CallOpSendMessage {
  public:
-  CallOpSendMessage() : send_buf_() {}
+  CallOpSendMessage() : send_message_called_, send_buf_() {}
```

Comment:
Why is send_message_called_ present above?

Reply:
Left it in by accident. Thanks for catching this!